CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+= Contributing to crd-ref-docs
```

Comment:
nit: markdownlint is complaining with the following messages:
> [MD041/first-line-heading/first-line-h1](https://github.com/DavidAnson/markdownlint/blob/v0.22.0/doc/Rules.md#md041): First line in file should be a top level heading

IIRC `=` is an alternative that should be used [below the text](https://www.markdownguide.org/basic-syntax/#alternate-syntax).